### DISCUSSION TITLE
New aircraft types

### DISCUSSION THREAD

**Discussion by nmeijers31** [junzis/openap#40] [General]
Hi @junzis,

Do you know if it is planned to add other aircraft types in the near future? Such as ATRs, Dash Q400, A220s, CRJs, Embraer E2s? Do you need ADS-B flight tracks only or do you require enhanced mode-S for these tracks? Happy to help on this if needed.
Many thanks for your fantastic job!

**Comment by junzis** [comment]
Yes, it is on my to-do list. The main thing missing is engine and emission models for these turboprops. For A220 and E2, they are easier once we have more ADS-B and Mode S data.
